coderadar-graph/src/main/java/io/reflectoring/coderadar/graph/contributor/repository/ContributorRepository.java

```diff
@@ -0,0 +1,27 @@
+package io.reflectoring.coderadar.graph.contributor.repository;
+
+import io.reflectoring.coderadar.graph.contributor.domain.ContributorEntity;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.data.neo4j.annotation.Query;
+import org.springframework.data.neo4j.repository.Neo4jRepository;
+import org.springframework.lang.NonNull;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ContributorRepository extends Neo4jRepository<ContributorEntity, Long> {
+  @Query("MATCH (c) WHERE ID(c) = {0} RETURN c")
+  @NonNull
+  Optional<ContributorEntity> findById(@NonNull Long id);
+
+  @Query("MATCH (c:ContributorEntity)-[:WORKS_ON]->(p:ProjectEntity) WHERE ID(p) = {0} RETURN c")
+  List<ContributorEntity> findAllByProjectId(Long projectId);
```

Comment:
Either add `@NonNull` or use `long`. (I've changed these all to just `long` as that is what a non-null long is)

Reply:
Done.